README.md

```diff
@@ -272,6 +272,12 @@ profile.
 
 - For `initscripts`,  `mac` is the currently configured MAC address of the device (`HWADDR`).
 
+### `match.path`
+The `match.path` is optional and restricts the profile to be usable only on devices
+with the given `match.path`. `match.path` is only allowed for type ethernet or infiniband
+to match a non-virtual device with the profile. Each element of `match.path` list is a shell
+wildcard pattern.
```

Comment:
This needs more explanation about what kind of path this is, how to determine which values are needed etc.